src/post.jl

```diff
@@ -140,6 +141,35 @@ end
 
 
 
+getlatency(c) = [l.latency for l in c.links]
+getlambda(c) = [l.ugn for l in c.links]
+getomega(t, xc) = [f(t) for f in xc.freq]
+getbeta(t, c, xc) = [beta(l, t, xc.theta) for l in c.links]
+getgamma(t, c, xc) = [gamma(l, t, xc.theta) for l in c.links]
```

Comment:
naming nit: these return arrays of latencies, ugns, etc., rather than a single latency, ugn, etc. Would it make it more clear by using plurals? getlatencies, getlamdas, or some other such sign that these operate on the entire graph/array.

Reply:
Even more of a nit, since these are local. And might be clear for Julia users already.